oops/oops.go

```diff
@@ -98,12 +132,12 @@ type stackWithReasons struct {
 	reasons []string
 }
 
-// Frames extracts all frames from an oops error. If err is not an oops error,
-// nil is returned.
-func Frames(err error) [][]Frame {
```

Comment:
This function is _almost_ un-callable though. It can only be called using output from this package, reflection, or some shenanigans. So maybe we can make the case that this is not really a breaking change. Feels worth calling out though and documenting why this is safe?

Reply:
This function is used within `Frames`. On GitHub it looks like it's being replaced, but `Frames` still exists with the same signature on line 233 in the diff.

Reply:
Heck. Reading is hard! Thanks!